lib/rotoscope.rb

```diff
@@ -8,14 +8,13 @@ class Rotoscope
   InvalidStateError = Class.new(StandardError)
 
   class << self
-    def trace(dest, blacklist: [], flatten: false, &block)
-      if flatten
-        return flat_trace(dest, blacklist, &block)
-      end
+    def trace(dest, options = {}, &block)
+      config = { blacklist: [], flatten: false }.merge(options)
```

Comment:
I'd encourage you to stick to explicitly calling out the supported kwargs (like you had before) and just construct the hash in here from the given args. It avoids any confusion (by raising `ArgumentError`) if someone were to do something like `trace("some_path", flaten: true)` (note the lack of the second `t`).